python/GafferSceneUITest/CameraToolTest.py

```diff
@@ -288,5 +290,55 @@ def testTransformNode( self ) :
 				)
 			)
 
+	def testEditScopes( self ) :
+
+		script = Gaffer.ScriptNode()
+		script["camera"] = GafferScene.Camera()
+
+		script["editScope"] = Gaffer.EditScope()
+		script["editScope"].setup( script["camera"]["out"] )
+		script["editScope"]["in"].setInput( script["camera"]["out"] )
+
+		view = GafferSceneUI.SceneView()
+		view["in"].setInput( script["editScope"]["out"] )
+
+		view["camera"]["lookThroughEnabled"].setValue( True )
+		view["camera"]["lookThroughCamera"].setValue( "/camera" )
+
+		tool = GafferSceneUI.CameraTool( view )
+		tool["active"].setValue( True )
+
+		script["camera"]["transform"]["translate"].setValue( imath.V3f( 1, 2, 3 ) )
+		script["camera"]["transform"]["rotate"].setValue( imath.V3f( 1, 2, 3 ) )
+
+		# Without EditScope, should edit Camera node directly.
+
+		cameraTransform = imath.M44f().translate( imath.V3f( 10, 5, 1 ) ).rotate( imath.V3f( math.pi / 4 ) )
+		view.viewportGadget().setCameraTransform( cameraTransform )
+
+		self.assertFalse( GafferScene.EditScopeAlgo.hasTransformEdit( script["editScope"], "/camera" ) )
+		self.assertTrue(
```

Comment:
Super-nit-pick-mode. Personally very much enjoying this all conforming to our new regime of 115 chars wide, so much easier to read on GitHub. I just noticed in the other test additions there is a varying degree of consistency in adhering to this. I guess old habits die hard :)